ajax/libs/clone/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "clone",
+  "filename": "clone.min.js",
+  "description": "deep cloning of objects and arrays",
+  "keywords": [
+    "clone",
+    "object",
+    "array",
+    "function",
+    "date"
+  ],
+  "version": "2.1.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pvorb/clone"
+  },
+  "author": "Paul Vorbach <paul@vorba.ch> (http://paul.vorba.ch/)",
+  "license": "MIT",
+  "npmName": "clone",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
@pvnr0082t sure I will change that. I wanted to ask if you could please tell me what the different between basePath: "src" and ""?